### PR TITLE
Fix: ControlBarrier rotates within LocalSpace

### PR DESCRIPTION
**Pull request by MarkusJohansen**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
The PR fixes a bug where the controlled barrier for switching between conveyour lanes, always rotated towards 45 degrees in worldspace instead of localspace. To ensure flexibility to the prefab, this has been changed to localspace, you can also reverse it, set the speed and set the amount of degrees you want to rotate towards.

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
Changes is made in the QA scene. It is also worth noting that the physics of the barrier collision is a bit buggy, and one a rare occation force is applied to fish in a direction that launches the fish in the air.

### CODE REVIEW

File: Assets/FishFactory/BarrierFishHandling.cs [hunk 1]
```diff
@@ -0,0 +1,22 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class BarrierFishHandling : MonoBehaviour
+{
+    // Start is called before the first frame update
+    [SerializeField] float slideForce = 10f; // You can adjust this value to get the desired sliding effect
```
Comment (jonp-h) [inline Assets/FishFactory/BarrierFishHandling.cs:8]: Serialized should be above with prettier?

**Review by jonp-h — CHANGES_REQUESTED** [review]
Did prettier stop working? Else good to merge

**Comment by jonp-h** [comment]
If a wall is moved to make the fish change conveyor belt, perhaps a lever instead of a button would be more logical? A button would make more sense for a hammer.

**Comment by MarkusJohansen** [comment]
> If a wall is moved to make the fish change conveyor belt, perhaps a lever instead of a button would be more logical? A button would make more sense for a hammer.

Sounds reasonable, however we should adress this when we find out wether we actually need something like a "hammer" to guide the fish to the correct path.

**Pull request merged**